scripts/rv_expression_association/prepare_inputs.py

```diff
@@ -0,0 +1,211 @@
+#!/usr/bin/env python3  # pylint: disable=missing-module-docstring
+
+import logging
+import subprocess
+import sys
+import re
+import click
+import pandas as pd
+from cloudpathlib import AnyPath
+from cpg_utils import to_path
+from cpg_utils.hail_batch import output_path
+
+subprocess.run(
+    [
+        sys.executable,
+        '-m',
+        'pip',
+        'install',
+        'limix==3.0.4',
+        'pandas_plink==2.2.9',
+        'xarray==0.20.2',
+    ],
+    check=True,
+)
+
+import xarray as xr  # pylint: disable=wrong-import-position, import-error
+from pandas_plink import (read_plink1_bin)  # pylint: disable=wrong-import-position, import-error
+from limix.qc import (quantile_gaussianize)  # pylint: disable=wrong-import-position, import-error
```

Comment:
```suggestion
# pylint: disable=wrong-import-position,import-error
import xarray as xr  
from pandas_plink import read_plink1_bin
from limix.qc import quantile_gaussianize
```

`A disable comment can be inserted on its own line, applying the disable to everything that comes after in the same block. Alternatively, it can be inserted at the end of the line for which it is meant to apply.`

Reply:
Right! I was going for option 2 here, should I just add it once at the top instead? I thought it made sense to only apply to relevant lines but maybe all the repetition is not great either?

Reply:
In this instance you're applying the same 2 disables to all 3 lines, and the disable only relates to imports, so it's pretty safe to include it once